### PR TITLE
Script creates a new calendar with custom start day for a user. 

### PR DESCRIPTION
**Pull request by wfletch**
it then saves this as a json file to disk

### CODE REVIEW

**Comment by wfletch** [comment]
@Bartocaster Please Review before merging.

File: create_new_calendar.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import sys, getopt
+import json
+
+
+days_of_week = ["Monday", "Tuesday", "Wednesday", "Thursday","Friday", "Saturday", "Sunday"]
+
+def get_next_day(current_day):
```
Comment (wfletch) [inline create_new_calendar.py:7]: What does this do!

File: create_new_calendar.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	else:
+		current_day_index +=1
+	return days_of_week[current_day_index]
+
+if __name__ == "__main__":
+	user = raw_input("Enter Name of user to create:\t\t")
+	first_day_of_year = raw_input("Enter first day of the year (01/01/21):\t")
+	if first_day_of_year not in days_of_week:
+		print("Invalid Start Day")
+		print(', '.join(days_of_week) + " are valid days")
```
Comment (Bartocaster) [inline create_new_calendar.py]: again whit the raw_input.
feels like i need to be more on top of this. i feel like i am missing som information.

File: create_new_calendar.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	cal = {}
+	current_day = first_day_of_year
+	while current_day_number < number_of_days:
+		if current_day == 'Monday':
+			current_week += 1
+		if current_week not in cal:
+			cal[current_week] = {}
+		cal[current_week][current_day] = {}
+		current_day_number+=1
+		current_day = get_next_day(current_day)
```
Comment (Bartocaster) [inline create_new_calendar.py]: this is nice i can see what its ment for, But i wish you would go a little more indept whit my on this.
this is nice black magic

File: create_new_calendar.py (hunk 2 above)
Comment (wfletch) [reply create_new_calendar.py]: I think raw_input got updated to just normal input.

File: create_new_calendar.py (hunk 3 above)
Comment (wfletch) [reply create_new_calendar.py:36]: Ok, I will add comments explaining what the code does.
Thank you for suggesting!

**Comment by Bartocaster** [comment]
well done

**Pull request merged**